crowdsec-docs/unversioned/beta_program.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: beta_program
+title: Beta Program
+sidebar_position: 1
+---
+<p align="center">
+<img src='/img/beta_program_header.png' alt="CrowdSec Beta Program" title="CrowdSec Beta Program" width="100%" />
+</p>
+
+## About the Beta Program
+
+Join our Beta program to get early access to new features and functionalities for the CrowdSec Security Engine and Console. Your participation and feedback are key to CrowdSec’s community-driven approach. 
+
+Help us refine and improve our offerings and provide solutions that serve you and the community better. 
+To join the CrowdSec Beta program, click the [Beta opt-in option directly in the Console](https://app.crowdsec.net/settings).
```

Comment:
Does this always direct to user settings? what if user is within an org?

Reply:
the org path has the org name in the path. 
This URL does take you to personal settings where you'll see the opt in section